model/client/admin/key/wallet_propose_request.go

```diff
@@ -10,3 +10,7 @@ type WalletProposeRequest struct {
 func (*WalletProposeRequest) Method() string {
 	return "wallet_propose"
 }
+
+func (*WalletProposeRequest) Validate() error {
+	return nil
```

Comment:
key_type is a required param 
Need to add the logic for: 
You must provide at most one of the following fields: passphrase, seed, or seed_hex
So cannot have more than one of them.

Reply:
Or ir does say you can run it with no params to create a random seed, so maybe don't need the required check on key_type